### DISCUSSION TITLE
Welcome to hospitalrun-frontend Discussions!

### DISCUSSION THREAD

**Discussion by jackcmeyer** [HospitalRun/hospitalrun-frontend#2508] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blestab** [comment]
Hi everyone 👋🏾 

Blessed here, part of the core maintenance team. I am passionate about healthcare and all things to do with making healthcare better for everyone through the use of technology.

**Comment by DrewGregory** [comment]
Hi!

I'm a part of [Stanford Code the Change](http://codethechange.stanford.edu/). We're delighted to help out with the project across all aspects of the codebase. I'm particularly excited to learn more about the new offline-first migration and to help out where I can.

**Reply by blestab** [reply]
Welcome @DrewGregory, looking forward to working with you and your team to #CodeTheChange and help make a difference in healthcare

**Reply by matteovivona** [reply]
cc @fox1t

**Comment by Meetumasih** [comment]
Hi 

I am new to GItHub. Can anyone please guide me on where to look for project documentation and how to report bugs?

**Comment by shabfactor** [comment]
hi all, i am new to this project. just wondering the login seems like broken as it always keep login by default as `HospitalRun Test` user and when i click logout it does not seems to work. Not sure if i am doing anything wrong? i have installed all dependencies and it seems to load the frontend and working fine locally.

**Reply by matteovivona** [reply]
there is an issue with cache and cookie invalidation. to change user try to delete these two

**Comment by ESteury** [comment]
We are an NGO (Kafakumba Clinic) opening a 7 bed clinic near Luanshya, Zambia and are interested in using this software as our primary hospital software system.  I know there's been some changes since it's inception, but wanted to know if anyone else has used it in such a manner and what was your experience?  
We would use the whole package from inventory, to lab, to patient records... I'm concerned as we will be using a staff who has little technical savy and I can honestly say I am limited as well.  Concerned about usability and glitches....Anyone have any thoughts?
Elinda Steury

**Comment by Md-Aashiqq** [comment]
Hey 
i Have one doubt , which typescript version can I use in this project,

**Reply by sanjayadhityaS** [reply]
hi Aashiqq 
did you find any suitable version for the project i am trying to do NPM install most dependencies are deprecated and legacy peer deps also gets me to error when i try to run
